script.js

```diff
@@ -0,0 +1,19 @@
+// defining a function which adds the "hidden" class to the target element
+// the class is defined elsewhere, in CSS
+function showNavModal() {
+    console.log("showsomething")
+    const navModalBtn = document.getElementById('navModal')
+    navModalBtn.classList.add('navModal-is-shown')
+}
+
+const hamburgerBtn = document.getElementById('hamburger')
```

Comment:
these couple of lines should go after the `hideNavModal` function, so that all of our functions are at the top (just a cleanup thing, doesn't change behavior)